src/views/virtualmachines/details/tabs/configuration/scripts/ScriptsTab.tsx

```diff
@@ -103,9 +104,11 @@ const ScriptsTab: FC<VirtualMachineScriptPageProps> = ({ obj: vm }) => {
               descriptionData={
                 <Stack hasGutter>
                   <div data-test="ssh-popover">
-                    <Trans ns="plugin__kubevirt-plugin" t={t}>
-                      <Text component={TextVariants.p}>Store the key in a project secret.</Text>
-                    </Trans>
+                    {isDynamicSSHInjection ? (
+                      <div>{t('Store the key in a project secret.')}</div>
```

Comment:
please use <></> instead of div
in general use React fragment instead of div if you just need a wrapper with no props:
https://www.knowledgehut.com/blog/web-development/understanding-react-fragments